docs/mkdocs/docs/api/macros/nlohmann_define_derived_type.md

```diff
@@ -48,14 +51,20 @@ friend void to_json(nlohmann::json&, const type&);
 friend void from_json(const nlohmann::json&, type&);
 ```
 
-Macros 3 and 4 add two functions to the namespace which take care of the serialization and deserialization:
+Macros 4 and 5 add two functions to the namespace which take care of the serialization and deserialization:
```

Comment:
Maybe also 6?

Reply:
`add two functions`
3 and 6 only add one
I have mentioned that one a little bit below this line:
`Macros 3 and 6 add one function to the namespace which take care of the serialization only:`